spec/lib/saml/ssoe_user_spec.rb

```diff
@@ -952,6 +1005,7 @@
           uuid: '85ba80dba1b93ed3bf080b2989cde313',
           email: nil,
           idme_uuid: nil,
+          logingov_uuid: nil,
           loa: { current: 3, highest: 3 },
           sign_in: {
             service_name: 'dslogon',
```

Comment:
Can we first change the context description of the test near this: `with missing ID.me UUID`, to `with missing ID.me UUID and missing login.gov UUID`
And then add a test: `with missing ID.me UUID but a defined login.gov uuid`, just to prove that this works with a login.gov uuid even if ID.me uuid is nil